github-cache/main.py

```
@@ -0,0 +1,133 @@
+
+if not exists(f"{BASE_PATH}.png"):
+    image = Image.new("RGBA", (1, 1))
+    save_image(image)
+
+async def fetch_contributors() -> list:
+    headers = {}
+    if GITHUB_API_KEY:
+        headers["Authorization"] = f"token {GITHUB_API_KEY}"
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get("https://api.github.com/repos/bjoern-buettner/roleplay-ai/contributors", headers=headers, timeout=aiohttp.ClientTimeout(total=30, connect=5)) as response:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)